src/IssueViz.Security/OpenInIDE/Http/ShowHotspotRequestHandler.cs

```diff
@@ -0,0 +1,117 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using System.ComponentModel.Composition;
+using System.Net;
+using System.Threading.Tasks;
+using Microsoft.Owin;
+using SonarLint.VisualStudio.Integration;
+using SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Contract;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security.OpenInIDE.Http
+{
+    /// <summary>
+    /// Handles low-level HTTP request to open a hotspot
+    /// </summary>
+    [Export(typeof(IOwinPathRequestHandler))]
+    internal class ShowHotspotRequestHandler : IOwinPathRequestHandler
+    {
+        private const string ParamName_Server = "server";
+        private const string ParamName_Project = "project";
+        private const string ParamName_Hotspot = "hotspot";
```

Comment:
I looked at the existing SQ API parameter names, and the standard form appears to be e.g. 'hotspot' rather than 'hotspotKey'.

There's an exception in the SQ API `api/hotspots/search` which has `projectKey`, but that appears to be an anomaly.